### ISSUE TITLE
Unable to use send_keys on input in nested shadow-root

### ISSUE THREAD

**Issue by mweinelt** [sukgu/pyshadow#5]
### Code

```py
from selenium.webdriver import Firefox
from selenium.webdriver.firefox.options import Options
from selenium.webdriver.support.ui import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC
from pyshadow.main import Shadow

options = Options()
options.add_argument("--headless")
driver = Firefox(options=options)
wait = WebDriverWait(driver, 10)

driver.get("http://localhost:8123/onboarding.html")
wait.until(EC.title_contains("Home Assistant"))

shadow = Shadow(driver)
name = shadow.find_element('input[name="name"]')
print(name)
name.send_keys("admin")
```

### Traceback
```pytb
QA--QAQA True
<selenium.webdriver.firefox.webelement.FirefoxWebElement (session="cb65f5b3-4b33-4188-bf1e-6a6135c655f8", element="99ed2527-07ab-4df9-bcc2-72556f7bdfee")>
Traceback (most recent call last):
  File "/run/current-system/sw/bin/test-runner", line 33, in <module>
    name.send_keys("admin")
  File "/nix/store/i70af0zjmlk62jcadrlj2ziwcfvz7ak4-python3-3.8.9-env/lib/python3.8/site-packages/selenium/webdriver/remote/webelement.py", line 477, in send_keys
    self._execute(Command.SEND_KEYS_TO_ELEMENT,
  File "/nix/store/i70af0zjmlk62jcadrlj2ziwcfvz7ak4-python3-3.8.9-env/lib/python3.8/site-packages/selenium/webdriver/remote/webelement.py", line 633, in _execute
    return self._parent.execute(command, params)
  File "/nix/store/i70af0zjmlk62jcadrlj2ziwcfvz7ak4-python3-3.8.9-env/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/nix/store/i70af0zjmlk62jcadrlj2ziwcfvz7ak4-python3-3.8.9-env/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.ElementNotInteractableException: Message: Element <input name="name"> is not reachable by keyboard
```

### DOM

DOM from the Firefox inspector tab below with the input tag.

![20210506_05h36m52s_grim](https://user-images.githubusercontent.com/131599/117238571-6bc89180-ae2d-11eb-942b-4b2ccf6a4444.png)

Image of the website with the name field marked red

### How the page looks

![image](https://user-images.githubusercontent.com/131599/117238767-c8c44780-ae2d-11eb-8287-249528867a69.png)

### Version

pyshadow 0.0.3
selenium 3.141.0
geckodriver 0.29.1
firefox 88.0.1

**Comment by sukgu** [comment]
`selenium.common.exceptions.ElementNotInteractableException Element <input name="name"> is not reachable by keyboard`
The above error implies that the input element is not interactable. Probably you can find there may be some wrapper elemnt which can be interactable, else try javascript executor as workaround to set the value directly.

**Comment by sukgu** [comment]
@mweinelt Also, please use the latest version v0.0.4 to check if this is fixed in v0.0.4.

**Comment by sukgu** [comment]
Please check this [link](https://stackoverflow.com/questions/49864965/org-openqa-selenium-elementnotinteractableexception-element-is-not-reachable-by).
